spec/requests/api/merge_requests_spec.rb

```diff
@@ -343,46 +346,87 @@
     it "should return merge_request with renamed target_branch" do
       put api("/projects/#{project.id}/merge_request/#{merge_request.id}", user), target_branch: "wiki"
       expect(response.status).to eq(200)
-      expect(json_response['target_branch']).to eq('wiki')
+      expect(json_response["target_branch"]).to eq("wiki")
     end
 
-    it 'should return 400 on invalid label names' do
+    it "should return 400 on invalid label names" do
       put api("/projects/#{project.id}/merge_request/#{merge_request.id}",
               user),
-          title: 'new issue',
-          labels: 'label, ?'
+          title: "new issue",
+          labels: "label, ?"
       expect(response.status).to eq(400)
-      expect(json_response['message']['labels']['?']['title']).to eq(['is invalid'])
+      expect(json_response["message"]["labels"]["?"]["title"]).to eq(["is invalid"])
```

Comment:
Line is too long. [84/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.